src/controller/task.ts

```diff
@@ -0,0 +1,104 @@
+import { Request, Response, NextFunction } from 'express'
+import TaskModel, { Task } from '../model/task'
+
+export default {
+  async createTask (req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+      const { title, description, importance, status } = req.body
+      const newTask: Task = await TaskModel.create({
+        title,
+        description,
+        importance,
+        status
+      })
+      res.status(201).send(newTask)
+    } catch (err) {
+      next(err)
+    }
+  },
+
+  async getAllTasks (req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+      const limit = parseInt(<string>req.query.limit)
+      const skip = parseInt(<string>req.query.skip)
+      const taskTotal: number = await TaskModel.countDocuments({})
+      const tasks: Task[] = await TaskModel.find({}).skip(skip).limit(limit).lean()
+      res.send({
+        tasks,
+        total: taskTotal,
+        limit: limit,
+        offset: skip
```

Comment:
Query param names should match those values. So `skip` name should be used in this case. (or offset if you change query param)